Gemfile.lock

```diff
@@ -170,7 +170,7 @@ GEM
     spring-watcher-listen (2.0.1)
       listen (>= 2.7, < 4.0)
       spring (>= 1.2, < 3.0)
-    sprockets (4.0.0)
+    sprockets (4.0.2)
```

Comment:
### [sprockets](https://github.com/rails/sprockets)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 4.0.0 → 4.0.2

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **41** commits. See the full changes on [the compare page](https://github.com/rails/sprockets/compare/v4.0.0...v4.0.2).

These are the first 10 commits:
- (e07cb42) [use a constant](https://github.com/rails/sprockets/commit/e07cb42a42f28d86eebee986aa7e44fa1c480167)
- (1cfc5a3) [Update ISSUE_TEMPLATE.md](https://github.com/rails/sprockets/commit/1cfc5a34fe23e5a6acc6315f1d77b7f9794fb766)
- (7d75883) [[ci skip] Document how rails precompile works](https://github.com/rails/sprockets/commit/7d7588364b428e35deb3d004057edf2520a9a477)
- (4da8e0f) [Fix typos in README](https://github.com/rails/sprockets/commit/4da8e0f67f25b3dab093970c8f1d799b81b7dbbc)
- (070fc01) [Merge pull request #634 from lagartoflojo/patch-1](https://github.com/rails/sprockets/commit/070fc01947c111d35bb4c836e9bb71962a8e0595)
- (606f841) [Merge pull request #629 from ahorek/constant](https://github.com/rails/sprockets/commit/606f841cd74888795b8745048ab54e04441865c7)
- (a1202aa) [Update ISSUE_TEMPLATE.md](https://github.com/rails/sprockets/commit/a1202aac84bd5352abd13c70c9aec37a1f448bed)
- (ba6fb1c) [mention Rails ependency restriction confusions in UPGRADING](https://github.com/rails/sprockets/commit/ba6fb1c292cdf9bb4e8ddfdeff170afa11d8e5a8)
- (9bcff52) [expand upon Rails sprockets 3 vs 4 default target logic in U…](https://github.com/rails/sprockets/commit/9bcff52c8aa911f0b9ef2ca237cb9c32a473ee57)
- (2a27675) [Additional README doc on link/link_tree/link_directory](https://github.com/rails/sprockets/commit/2a27675d9de93a35b455c520d5bdbfb34dd88fd7)


</details>